tests/foreman/cli/test_syncplan.py

```diff
@@ -365,6 +363,33 @@ def test_positive_info_enabled_field_is_displayed(self):
         result = SyncPlan.info({'id': new_sync_plan['id']})
         self.assertIsNotNone(result.get('enabled'))
 
+    @tier1
+    @upgrade
+    def test_positive_info_with_assigned_product(self):
+        """Verify that sync plan info command returns list of products that
+        assigned to that sync plan
+
+        :id: 7a7e5e40-7776-4d26-9173-73f00de6e8e9
+
+        :expectedresults: Expected product information is returned in info
+            command
+
+        :BZ: 1390545
+        """
+        sync_plan = self._make_sync_plan({
+            'enabled': 'false',
+            'organization-id': self.org['id'],
+            'sync-date': datetime.utcnow().strftime("%Y-%m-%d %H:%M:%S"),
+        })
+        product = make_product({'organization-id': self.org['id']})
+        Product.set_sync_plan({
+            'id': product['id'],
+            'sync-plan-id': sync_plan['id'],
+        })
+        updated_plan = SyncPlan.info({'id': sync_plan['id']})
+        self.assertEqual(len(updated_plan['products']), 1)
```

Comment:
Maybe it's worth checking whether a _list_, not just 1 field, is returned? You could assign 2 products and make sure both are present in output.